### PR TITLE
fix typos in docstrings

### PR DESCRIPTION
**Pull request by discdiver**
## Summary

Fixes a few typos in ECSTask docstring.

## Checklist
- [ x] Summarized PR's changes in the **Unreleased** section of the [CHANGELOG.md](https://github.com/PrefectHQ/prefect-aws/blob/main/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 
 ### Fixed
+Typos in ECSTask docstring.
```
Comment (ahuang11) [inline CHANGELOG.md:19]: Perhaps this doesn't need to be in CHANGELOG
```suggestion
```

**Review by ahuang11 — CHANGES_REQUESTED** [review]

**Comment by discdiver** [comment]
lol. Agree. Maybe the PR template should specify the Changelog is optional
- now it wants a checkmark.

**Comment by ahuang11** [comment]
I agree

**Comment by ahuang11** [comment]
Would you like me to wrap this up for you?

**Comment by discdiver** [comment]
Thanks, Andrew. I didn't see you had made the change. I committed it.

**Review by ahuang11 — APPROVED** [review]

**Pull request merged**